### ISSUE TITLE
Bug with boolean parameters

### ISSUE THREAD

**Issue by yzh44yzh** [balance-platform/pillar#57]
Hello. I found out that pillars incorrectly converts boolean parameters.

Example:
```
iex(4)> ClickhouseMaster.query("select 1 where 1 == 1");
{:ok, 1}

iex(5)> ClickhouseMaster.query("select 1 where 1 == {bool_value}", %{bool_value: true})
{:error,
 %Pillar.HttpClient.Response{
   body: <<67, 111, 100, 101, 58, 32, 54, 50, 46, 32, 68, 66, 58, 58, 69, 120,
     99, 101, 112, 116, 105, 111, 110, 58, 32, 83, 121, 110, 116, 97, 120, 32,
     101, 114, 114, 111, 114, 58, 32, 102, 97, 105, 108, 101, 100, 32, 97,
     ...>>,
   headers: [
```

The problem is here: https://github.com/balance-platform/pillar/blob/master/lib/pillar/query_builder.ex#L15-L17

```
iex(7)> String.replace("where 1 == {bool_value}", ["{bool_value}"], fn _ -> 1 end)
<<119, 104, 101, 114, 101, 32, 49, 32, 61, 61, 32, 1>>
```

The fix is that
https://github.com/balance-platform/pillar/blob/master/lib/pillar/type_convert/to_clickhouse.ex#L26
should return string, not int, as it is done here:
https://github.com/balance-platform/pillar/blob/master/lib/pillar/type_convert/to_clickhouse.ex#L22

By the way, `convert` could be implemented like this:
```
def convert(true), do: "1"
def convert(false), do: "0"
```
but that is not a big deal.

**Comment by sofakingworld** [comment]
@yzh44yzh Hello! 

I confirm the bug you found, thanks for research!
Would you like make PR to contribute the fix above?

**Comment by yzh44yzh** [comment]
ok, I will do it.

**Comment by sofakingworld** [comment]
Published version 0.34.1, which includes fix
